modules/cnf-configuring-workload-hints.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc
+
+:_content-type: CONCEPT
+[id="cnf-configuring-workload-hints_{context}"]
+= Configuring workload hints
+
+Workload hints simplify the fine-tuning of performance to industry sector settings. Instead of a “one size fits all” approach, workload hints can cater to usage patterns such as placing priority on:
+
+* Low latency
+* Real-time capability
+* Efficient use of power
+
+In an ideal world, all of those would be prioritized: in real life, some come at the expense of others. The Node Tuning Operator is now aware of the workload class and better able to meet the demands of a workload class. The cluster admin can now specify into which use case a workload falls. The Node Tuning Operator uses hints fine tune the performance settings.
```

Comment:
```suggestion
In an ideal world, all of those would be prioritized: in real life, some come at the expense of others. The Node Tuning Operator is now aware of the workload expectations and better able to meet the demands of the workload. The cluster admin can now specify into which use case his workload falls. The Node Tuning Operator uses hints to fine tune the performance settings for it.
```

Reply:
Done with minor style guide changes - such as "his" -> "that".